src/main/scala/com/amazon/deequ/checks/Check.scala

```diff
@@ -860,16 +861,121 @@ case class Check(
       includeUpperBound: Boolean = true,
       hint: Option[String] = None)
     : CheckWithLastConstraintFilterable = {
+      isContainedInRange(column, lowerBound, upperBound, includeLowerBound, includeUpperBound, hint)
+  }
 
+  /**
+    * Asserts that the non-null values in a column fall into the predefined interval
+    *
+    * @param column column to run the assertion
+    * @param lowerBound lower bound of the interval
+    * @param upperBound upper bound of the interval
+    * @param includeLowerBound is a value equal to the lower bound allowed?
+    * @param includeUpperBound is a value equal to the upper bound allowed?
+    * @param hint A hint to provide additional context why a constraint could have failed
+    * @param quoteBounds should the bound values be quoted? Quoting is required for data
+             types such as DateType, TimestampType, StringType.
+    */
+  private def isContainedInRange[T](
+      column: String,
+      lowerBound: T,
+      upperBound: T,
+      includeLowerBound: Boolean,
+      includeUpperBound: Boolean,
+      hint: Option[String] = None,
+      quoteBounds: Boolean = false)
+  : CheckWithLastConstraintFilterable = {
     val leftOperand = if (includeLowerBound) ">=" else ">"
```

Comment:
Please use Spark's functions to programmatically build the query rather than building it yourself using string concatenation.